packages/pieces/discourse/src/lib/actions/new-topic.action.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { HttpMethod, httpClient } from '@activepieces/pieces-common';
+import { discourseAuth } from '../../index';
+import { Property, createAction } from "@activepieces/pieces-framework";
+
+
+
+export const newTopic = createAction({
+    auth: discourseAuth,
+    name: 'new_topic',
+    description: 'Create a new topic in Discourse',
+    displayName: 'New Topic',
```

Comment:
Can you please change to "Create Topic", It's better to use verb with the action

Reply:
done